runtime/tests/checker/entitlements_test.go

```diff
@@ -6080,7 +6080,7 @@ func TestCheckIdentityMapping(t *testing.T) {
 
                 init() {
                     let x = X()
-                    self.x1 = &x as auth(A, B, C) &X
```

Comment:
Would it be possible to keep the existing test and only adjust the assertion (if needed), and add the modified version as a new test?